package.json

```diff
@@ -15,21 +15,32 @@
     "posttest": "npm run lint && npm run test:git-history"
   },
   "devDependencies": {
+    "@babel/cli": "^7.2.3",
     "@babel/core": "^7.2.2",
     "@babel/polyfill": "^7.2.5",
     "@commitlint/cli": "^8.2.0",
     "@commitlint/config-conventional": "^8.2.0",
-    "amex-jest-preset-react": "^5.0.1",
-    "enzyme": "^3.9.0",
-    "enzyme-to-json": "^3.2.2",
+    "amex-jest-preset-react": "^4.0.0",
```

Comment:
why was this downgraded?

Reply:
@10xLaCroixDrinker Thank you, upgrading this

Reply:
9b1abbb updated